packages/trader/src/App/Components/Elements/ContractDrawer/contract-drawer.jsx

```diff
@@ -37,6 +38,7 @@ const ContractDrawer = observer(
         toggleHistoryTab,
     }) => {
         const { common, ui } = useStore();
+        const { symbol } = useTraderStore();
```

Comment:
@akmal-deriv this symbol is not used in ContractDetails, is it? :)

Reply:
Yeah, it can actually be removed 👍🏻